pandas/__init__.py

```diff
@@ -211,6 +210,19 @@ class Panel:
                 pass
 
             return Panel
+
+        elif name == "np":
```

Comment:
this only works for py37+, py36 needs to be handled separately

Reply:
@jbrockmendel side issue - we should remove the Panel compat

Reply:
added py36 deprecation.